src/main/java/com/yahoo/bullet/parsing/LeafExpression.java

```diff
@@ -19,8 +19,12 @@
 
 @Getter @Setter
 public class LeafExpression extends Expression {
+    public static final BulletError LEAF_EXPRESSION_REQUIRES_VALUE_FILED_ERROR =
+            makeError("The expression needs a value field", "Please provide a value field.");
     public static final BulletError LEAF_EXPRESSION_REQUIRES_PRIMITIVE_TYPE_ERROR =
-            makeError("The LeafExpression needs a value of primitive type", "Please provide a value of primitive type.");
+            makeError("The expression needs a value of primitive type", "Please provide a value of primitive type.");
```

Comment:
"The expression needs type set to only a primitive type", "Please provide a primitive type"

Reply:
Done.